bin/check-disk-usage.rb

```diff
@@ -130,6 +134,7 @@ def fs_mounts
         next if config[:ignoremnt] && config[:ignoremnt].include?(line.mount_point)
         next if config[:ignorepathre] && config[:ignorepathre].match(line.mount_point)
         next if config[:ignoreopt] && config[:ignoreopt].include?(line.options)
+        next if config[:ignorereadonly] && line.options.split(',').include?('ro')
```

Comment:
It makes sense I had to double check the man page to make sure that the option for remounting a panicked filesystem as read only is `remount-ro` and should not cause an issue.
```
errors={panic|continue|remount-ro}
              Specify  FAT behavior on critical errors: panic, continue without doing anything, or remount the parti‐
              tion in read-only mode (default behavior).
```